packages/features/bookings/lib/timePreferences.ts

```diff
@@ -0,0 +1,34 @@
+import { create } from "zustand";
```

Comment:
Tiny hook and store that makes the time settings super accessible and easy to use <3